src/controllers/UserController.js

```diff
@@ -4,10 +4,15 @@ import db from '../models';
 export const signUp = async (req, res) => {
   try {
     const user = await db.User.create(req.body);
-    res.status(201).json({ user });
+    res.status(201).json({
+      user
+    });
   } catch (err) {
     const status = 400;
     const error = err.errors[0].message;
-    res.status(status).json({ status, error });
+    res.status(status).json({
+      status,
+      error
+    });
   }
```

Comment:
update your branch. I don't think theses guys are in the userController file anymore.